napari/layers/intensity_mixin.py

```diff
@@ -39,8 +39,13 @@ def __init__(self, *args, **kwargs) -> None:
         self._gamma = 1
         self._colormap_name = ''
         self._contrast_limits_msg = ''
-        self._contrast_limits = [None, None]
-        self._contrast_limits_range = [None, None]
+        self._contrast_limits: Tuple[Optional[float], Optional[float]] = (
+            None,
+            None,
+        )
+        self._contrast_limits_range: Tuple[
+            Optional[float], Optional[float]
+        ] = [None, None]
```

Comment:
Declared as a tuple, but then assigned to a list.

```suggestion
        ] = (None, None)
```

More generally, it's unfortunate that the private attribute is a tuple, but then the returned property is a list. Maybe there's a need for mutability of the external property, but that may not actually work as expected if it's just a view of the immutable tuple underneath. I know your changes didn't introduce that consistency, but might be worth considering here.

Reply:
This still needs to be fixed.

Reply:
Confused at what GitHub is up to here, pretty sure this is fixed on the current set of commits 😄

Reply:
Oh woops my bad, was looking at the wrong line...

Reply:
Should be fixed now